### PR TITLE
Add action doc

### PR DESCRIPTION
**Pull request by taojintianxia**
Changes proposed in this pull request:
  - add Chinse and English version code of conduct for GitHub Action

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: docs/community/content/involved/conduct/code.en.md [hunk 1]
```diff
@@ -108,3 +108,14 @@ The following code of conduct is based on full compliance with [ASF CODE OF COND
    - If a rule's branch is over than `5`, every branch take a new line.
    - Rule name of parser should same with java variable's camel case.
    - Define separate files for every SQL type, file name should consist of `database` + `SQL type` + `Statement`. For example: `MySQLDQLStatement.g4`.
+
+## GitHub Action 规范
+
+- Workflow file name must end with `.yml`
+- Workflow file name must consist with the lowercase of `trigger type-action type`, for example: `nightly-check.yml`. omit trigger type for pull_request, for example: `check.yml`
```
Comment (terrymanu) [inline docs/community/content/involved/conduct/code.en.md:115]: It is better to rewrite `trigger type-action type` to `triggerType-actionType`

File: docs/community/content/involved/conduct/code.en.md [hunk 2]
```diff
@@ -108,3 +108,14 @@ The following code of conduct is based on full compliance with [ASF CODE OF COND
...
+- Execution action includes: check, ci, e2e, build
```
Comment (terrymanu) [inline docs/community/content/involved/conduct/code.en.md:116]: Action type or Execution action?

File: docs/community/content/involved/conduct/code.en.md [hunk 3]
```diff
@@ -108,3 +108,14 @@ The following code of conduct is based on full compliance with [ASF CODE OF COND
...
+- Trigger type includes: pull_request (without prefix), nightly
```
Comment (terrymanu) [inline docs/community/content/involved/conduct/code.en.md:117]: Trigger type should be the 1st line, Action type 2nd, just keep same sequence with name

File: docs/community/content/involved/conduct/code.en.md [hunk 4]
```diff
@@ -108,3 +108,14 @@ The following code of conduct is based on full compliance with [ASF CODE OF COND
    - If a rule's branch is over than `5`, every branch take a new line.
    - Rule name of parser should same with java variable's camel case.
    - Define separate files for every SQL type, file name should consist of `database` + `SQL type` + `Statement`. For example: `MySQLDQLStatement.g4`.
+
+## GitHub Action 规范
```
Comment (terrymanu) [inline docs/community/content/involved/conduct/code.en.md:112]: Translate?

Comment (terrymanu) [inline docs/community/content/involved/conduct/code.en.md:112]: All English conducts are end with `.`, please do same thing on `GitHub Action Conduct` as well.

Comment (taojintianxia) [reply docs/community/content/involved/conduct/code.en.md:112]: done

File: docs/community/content/involved/conduct/code.en.md (hunk 1 above)
Comment (taojintianxia) [reply docs/community/content/involved/conduct/code.en.md:115]: done

File: docs/community/content/involved/conduct/code.en.md (hunk 2 above)
Comment (taojintianxia) [reply docs/community/content/involved/conduct/code.en.md:116]: done

File: docs/community/content/involved/conduct/code.en.md (hunk 3 above)
Comment (taojintianxia) [reply docs/community/content/involved/conduct/code.en.md:117]: done

**Review by terrymanu — APPROVED** [review]

**Pull request merged**